numpy/ma/extras.py

```diff
@@ -2056,9 +2048,6 @@ def flatnotmasked_contiguous(a):
     slice_list : list
         A sorted sequence of `slice` objects (start index, end index).
 
-        .. versionchanged:: 1.15.0
-            Now returns an empty list instead of None for a fully masked array
-
```

Comment:
This comment should be worked into the Returns section. Can you suggest a wording?